tools/codegen/dest/register_dispatch_key.py

```diff
@@ -84,15 +84,34 @@ def gen_resize_out_helper(backend_index: BackendIndex) -> List[str]:
     } else if (options.memory_format_opt().has_value()) {
       out.unsafeGetTensorImpl()->empty_tensor_restride(*options.memory_format_opt());
     }
+  } else {
+    at::assert_no_internal_overlap(out);
```

Comment:
This is failing you in tests and you're not allowed to do it anyway, because some operators are OK with internal overlap and structured doesn't know which ones are which.